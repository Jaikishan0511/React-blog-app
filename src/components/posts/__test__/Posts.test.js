import React from "react";
import renderer from "react-test-renderer";

import Posts from "../index";
describe("Posts", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Posts />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
