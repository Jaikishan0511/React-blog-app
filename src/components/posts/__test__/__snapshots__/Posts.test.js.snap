// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts snapshot renders 1`] = `
<div>
  <div />
</div>
`;
